dellyGermline.wdl

```diff
@@ -168,9 +169,15 @@ workflow dellyGermline {
       }
     ]
     output_meta: {
-      germlineSVs: "Filtered vcf file containing PASS structural variant calls",
-      germlineCNVs: "Filtered vcf file containing copy number variant calls"
+    germlineSVs: {
+        description: "Filtered vcf file containing PASS structural variant calls",
```

Comment:
My mistake, this should not say "PASS" but rather: "Filtered vcf file containing structural variant calls"

Reply:
fixed